src/naming.jl

```diff
@@ -36,6 +36,9 @@ See also [`parse_savename`](@ref).
 * `accesses = allaccess(c)` : You can also specify which specific keys you want
   to use with the keyword `accesses`. By default this is all possible
   keys `c` can be accessed with, see [`allaccess`](@ref).
+* `ignores = allignore(c)` : You can also specify which specific keys you want
+  to ignore with the keyword `ignores`. By default this is an empty 
+  tuple, see [`allignore`](@ref).
```

Comment:
Please add that `ignores` overwrites `accesses`

Reply:
Added.